public_html/wp-content/plugins/camptix-attendance/addons/assets/attendance-ui.js

```diff
@@ -421,8 +426,9 @@ jQuery(document).ready(function($){
 
 			this.render();
 
-			this.$header = this.$el.find( 'header' );
-			this.$menu = this.$header.find( '.menu' );
+			this.$header = this.$el.find('header');
+			this.$qrscanner = this.$el.find('.qr-scanner');
+			this.$menu = this.$header.find('.menu');
```

Comment:
The property `this.$qrscanner` is never used, so the changes in these lines can be reverted.